tests/management_api_tests/endpoints/test_endpoints.py

```diff
@@ -340,9 +358,12 @@ def test_success_to_create_second_endpoint_with_max_endpoints_gt_one(
     body['spec']['resources'] = ENDPOINT_RESOURCES
     url = ENDPOINTS_MANAGEMENT_API_URL.format(tenant_name=namespace)
     response = requests.post(url, data=json.dumps(body['spec']), headers=DEFAULT_HEADERS)
+    endpoint_url = get_url_from_response(response)
     assert response.status_code == 200
-    assert "Endpoint created" in response.text
-    assert f"{endpoint_name}-{namespace}" in response.text
+    assert {'status': 'CREATED',
+            'data': {'url': endpoint_url,
+                     'warning': '{} model is not available on the platform'.format(
+                         body['spec']['modelName'])}} == json.loads(response.text)
```

Comment:
Big block, used about three times. Is it possible to extract it? Would it be more readable?